src/back/controllers/api/routes/categories/categories.js

```diff
@@ -9,10 +10,30 @@ const getAll = (req, res) => {
 };
 
 const save = (req, res) => {
-    // TODO: Implement save categories functionality
-    res.sendStatus(200);
+    const userId = req.user._id;
+    const categoryIds = getCategoryIds(req.body);
+
+    user.find({ _id: userId }).update({
```

Comment:
We've talked about updating user by his `id` to avoid making two requests to DB. In this case you still have two requests: `user.find()` and `.update()`. So it is better either use `req.getCurrentUser()` (it still does two requests, but at least abstracts itself from storage) or make one update request like [here](http://mongoosejs.com/docs/documents.html): `Tank.update({ _id: id }, { $set: { size: 'large' }}, callback)`.